src/roles/common/tasks/check-local-file-present.yml

```diff
@@ -0,0 +1,12 @@
+- name: Check if file is present on localhost
+  stat:
+    path: "{{ file_path }}"
+  register: file_present
+  delegate_to: localhost
+
+- name: Assert that file must exist on localhost
+  assert:
+    that: "file_present.stat.exists"
+    success_msg: "{{ success_message | default('File exists') }}: {{ file_path }}"
+    fail_msg: "{{ fail_message | default('File does not exist') }}: {{ file_path }}"
+  ignore_errors: "{{ continue_on_error | default(false)}}"
```

Comment:
Add a newline at the end. In general it's a good idea to check for No newline at the end of file error in github commit before uploading a PR. It's the red circle with a red line in between that indicates the newline is missing.